core/teamserver/modules/boo/execute-assembly.py

```diff
@@ -8,15 +8,26 @@ class STModule(Module):
     def __init__(self):
         self.name = 'boo/execute-assembly'
         self.language = 'boo'
-        self.description = 'Execute a .NET assembly in memory'
-        self.author = '@byt3bl33d3r'
+        self.description = 'Execute local or remote .NET assembly in memory. Remote-project uses Msbuild to load *.xml or *.csproj projects'
+        self.author = '@byt3bl33d3r, @anthemtotheego (Csharp project), @hackabean(BooLang port)'
         self.references = []
         self.options = {
-            'Assembly': {
-                'Description'   :   'Path to assembly',
+            'Local-Assembly': {
+                'Description'   :   'Path to local assembly',
                 'Required'      :   True,
                 'Value'         :   ''
```

Comment:
@hackabean seems like the tests are failing because there isn't a default value here. Can you take a look at the module tests and set it manually through there? 
Thanks!

Reply:
The only error that appears comes from: 
``
2019-10-04 21:41:20,689 2998 MainThread - [ERROR] loader.py: get_loadables - Failed loading listener core/teamserver/listeners/wmi.py: No module named 'impacket'
``
Which as far as I have tested does not prevent ST from running correctly.

Reply:
@byt3bl33d3r I added default values and still nothing. Teamserver is loading modules without any errors.

Reply:
@hackabean no worries, i'll take a look later today.
Cheers